umami.libraries.yml

```diff
@@ -5,6 +5,8 @@ global:
       css/base.css: {}
       css/components/blocks/branding/branding.css: {}
       css/components/blocks/search/search.css: {}
+      css/components/fields/label-items.css: {}
+      css/components/listing-pages/recipes/listing-pages-recipes.css: {}
```

Comment:
Shouldn't this be added to the layout folder instead of components/listing-pages?

Reply:
Sorry Thamas, I followed the existing structure here for articles and the only layout directory we currently have is in the templates directory where we have the twig unless I've missed something here. Since articles are already here, shall we fix this under a separate issue?